### ISSUE TITLE
Support for etcd v3

### ISSUE THREAD

**Issue by lucastheisen** [kelseyhightower/confd#574]
It appears etcd v3 is not yet supported:

```
# confd -onetime -backend etcd -node=http://172.17.0.2:2379
2017-06-23T16:11:17Z be2621c9ebce confd[27]: INFO Backend set to etcd
2017-06-23T16:11:17Z be2621c9ebce confd[27]: INFO Starting confd
2017-06-23T16:11:17Z be2621c9ebce confd[27]: INFO Backend nodes set to http://172.17.0.2:2379
2017-06-23T16:11:20Z be2621c9ebce confd[27]: ERROR client: etcd cluster is unavailable or misconfigured
2017-06-23T16:11:20Z be2621c9ebce confd[27]: FATAL client: etcd cluster is unavailable or misconfigured
```

Any plan to add support for the new [gRPC](https://github.com/coreos/etcd/blob/master/Documentation/dev-guide/api_reference_v3.md) api?

**Comment by lucastheisen** [comment]
This appears to have been addressed by pull request #468...  Just waiting on @kelseyhightower  to merge...  Gonna see if the @bacongobbler [0.12.1 branch works](https://github.com/bacongobbler/confd/releases/tag/v0.12.1) in the meantime...

**Comment by lucastheisen** [comment]
Duplicate of #544

**Comment by wi-cuckoo** [comment]
So, how to use etcd client v3

**Comment by bacongobbler** [comment]
see https://github.com/kelseyhightower/confd/pull/468.
